web/vasilemaftita/css/css-home/csshome.css

```diff
@@ -0,0 +1,14 @@
+h1 {
+  background-color: blue;
+}
+body {
+  background-color: lightblue;
+}
+p {
+  font-family: verdana;
+  font-size: 20px;
+  color: brown;
+}
+p {
+  class="p3";
```

Comment:
The way you should be targeting a class from your index.html is using `.` . So delete lines 12-14 and replace them with `.p3 {
  /* style here the class .p3 */
}`